app/controllers/tags_controller.rb

```diff
@@ -163,7 +163,7 @@ def list
       @list.prev_topics_url = construct_url_with(:prev, list_opts)
       @rss = "tag"
       @title = I18n.t("rss_by_tag", tag: tag_params.join(" & "))
-      @description_meta = @tag.where(name: @tag_id).pick(:description) || @title
+      @description_meta = Tag.find_by_name(@tag_id)&.description || @title
```

Comment:
Sorry my previous suggestion should be `Tag.where(name: @tag_id).pick(:description)`.